docs/component_APIs/area_props.md

```diff
@@ -0,0 +1,170 @@
+
+All color properties use the CSS color syntax. Examples: "red", "#070707", "#222", "rgba(255, 255, 0, 1)", "hsl(0, 100%, 50%)".
+
+* [fill](#fill)
+* [fillOpacity](#fillopacity)
+* [stroke](#stroke)
+* [strokeOpacity](#strokeopacity)
+* [strokeWidth](#strokewidth)
+* [strokeLinecap](#strokelinecap)
+* [strokeLinejoin](#strokelinejoin)
+* [strokeMiterlimit](#strokemiterlimit)
+* [transform](#transform)
+
+## fill
+
+The fill color for the component.
+
+| **Type**       | **Required** | **Default** |
+| -------------- | ------------ | ----------- |
+| string (Color) | false        | 'none'      |
+
+## fillOpacity
+
+The opacity of the fill (between 0 and 1). Gets multiplied with the fill colors alpha value.
+
+| **Type**                  | **Required** | **Default** |
+| ------------------------- | ------------ | ----------- |
+| number \| string (number) | false        | 1           |
+
+## stroke
+
+The stroke (line) color for the component.
+
+| **Type**       | **Required** | **Default** |
+| -------------- | ------------ | ----------- |
+| string (Color) | false        | 'none'      |
+
+## strokeOpacity
+
+The opacity of the stroke (between 0 and 1). Gets multiplied with the stroke colors alpha value.
+
+| **Type**                  | **Required** | **Default** |
+| ------------------------- | ------------ | ----------- |
+| number \| string (number) | false        | 1           |
+
+## strokeWidth
+
+| **Type**                  | **Required** | **Default** |
+| ------------------------- | ------------ | ----------- |
+| number \| string (number) | false        | 1           |
+
+## strokeLinecap
+
+| **Type**                        | **Required** | **Default** |
+| ------------------------------- | ------------ | ----------- |
+| enum('flat', 'round', 'square') | false        | 'flat'      |
+
+## strokeLinejoin
+
+| **Type**                        | **Required** | **Default** |
+| ------------------------------- | ------------ | ----------- |
+| enum('miter', 'round', 'bevel') | false        | 'miter'     |
+
+## strokeMiterlimit
+
+How far to extend the stroke at a sharp corner when using `strokeLinejoin='miter'`, see [here](https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/stroke-miterlimit) for a more detailed explanation.
+
+| **Type**                  | **Required** | **Default** |
+| ------------------------- | ------------ | ----------- |
+| number \| string (number) | false        | 10          |
+
+## transform
+
+List of transformations to apply to the component (are quite similar to SVG transformations). Example for multiple transformations: `transform="translate(100, 100) rotate(90)"`.
+All x and y coordinates specified in a transformation are relative _to the component itself_, meaning that `translate(-50%, 0)` will translate the component by 50% of it's own width to left.
+
+| **Type** | **Required** | **Default** |
+| -------- | ------------ | ----------- |
+| string   | false        | ''          |
+
+### rotate
```

Comment:
Isn't added to the list of props at the top.

Reply:
Added